src/main/java/nl/avans/drivio/model/Car.java

```diff
@@ -1,138 +1,129 @@
 package nl.avans.drivio.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.Collection;
 
 @Entity
 @Table
 public class Car {
     @Id
-    @SequenceGenerator(
-            name = "car_sequence",
-            sequenceName = "car_sequence",
-            allocationSize = 1
-    )
-    @GeneratedValue(
-            strategy = GenerationType.SEQUENCE,
-            generator = "car_sequence"
-    )
-    private int CarId;
-    private String Brand;
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "car_id")
+    private Integer carId;
+    private String brand;
     private String Model;
-    private String FuelType;
-    private int BuildYear;
-    private String NumberPlate;
-    private String CarType;
-    private String GearBox;
-//    private User Owner;
-//    private User Renter;
+    private String fuelType;
+    private int buildYear;
+    private String numberPlate;
+    private String carType;
+    private String gearBox;
+
```

Comment:
Hier mist de koppeling met de user tabel. Hier kunnen we het beste mijn code gebruiken ipv dit bestand. De car ratings kunnen er dan altijd nog aan toegevoegd worden. Mee eens?

Reply:
Fixed met merge conflicts in commit https://github.com/leonvsc/Drivio/pull/26/commits/d6c224c192476645187e85243bfa37e53c615ef4